config/config.php

```diff
@@ -110,66 +110,66 @@
 	'stable' => [
 		'28' => [
 			'100' => [
-				'latest' => '28.0.2',
-				'internalVersion' => '28.0.2.5',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-28.0.2.zip',
+				'latest' => '28.0.3',
+				'internalVersion' => '28.0.3.2',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-28.0.3.zip',
```

Comment:
should be `releases`

Reply:
You haven't read the first post, haven't you? :grin: